test/src/api/expect/testExpectCookie.js

```diff
@@ -320,27 +320,6 @@ xdescribe('expect.cookie', function() {
     });
   });
 
-  it('with domain to equal - multiple cookies found [FAILED]', function() {
```

Comment:
this still looks like a relevant test, why was it removed?

Reply:
Yes, it was the relevant tests, but in https://github.com/vaibhavsingh97/nightwatch/blob/72b8ed43c0e3287821e9301d9271500ab1387d7d/lib/transport/selenium-webdriver/method-mappings.js#L555-L559, we are returning only a single cookie when we fetch cookie by name

Ref: https://github.com/SeleniumHQ/selenium/blob/68f3edcb9b28925757d3f19ba2a255d7d2d6b5b6/javascript/node/selenium-webdriver/lib/webdriver.js#L1672-L1676

Reply:
Ok, sounds good.